src/sql/workbench/contrib/editData/browser/gridParentComponent.ts

```diff
@@ -501,10 +501,14 @@ export abstract class GridParentComponent extends Disposable {
 	resizeGrids(): void {
 		const self = this;
 		setTimeout(() => {
-			for (let grid of self.renderedDataSets) {
-				grid.resized.fire();
-			}
-		});
+			self.onResize();
```

Comment:
Shouldn't need self here either 

Arrow lambdas preserve this.

Explanation: 
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-this-alias.md